client/packages/host/src/components/Footer/UserDetails.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from 'react';
+import {
+  Box,
+  CircularProgress,
+  FlatButton,
+  PaperPopoverSection,
+  useAuthContext,
+  usePaperClickPopover,
+  useTranslation,
+  useNavigate,
+  useUserDetails,
+  useConfirmationModal,
+  RouteBuilder,
+  PowerIcon,
+  TextWithLabelRow,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+import { PropsWithChildrenOnly } from '@common/types';
+
+export const AdminSelector: FC<PropsWithChildrenOnly> = ({ children }) => {
```

Comment:
Naming request from me... can we call this `UserDetails`, like the file is called?

Selector makes sense for other popups (selecting a different store or language) but less so here 🙏